mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/ProcessSourcesMojo.java

```diff
@@ -49,6 +49,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         getContentGenerator().createApplicationClassLoaderModelJsonFile(classLoaderModel);
       } catch (Exception e) {
         getLog().debug(format("There was an exception while creating the repository of [%s]", project.toString()), e);
+        throw new MojoFailureException(e.getMessage(), e);
```

Comment:
can you please change the exception to 
String message = format("There was an exception while creating the repository of [%s]", project.toString());
getLog().debug(message, e);
throw new MojoFailureException(message, e);